### PR TITLE
dev-libs/libmodbus: bump up to 3.1.6

### PR DESCRIPTION
**Pull request by l29ah**

### CODE REVIEW

File: dev-libs/libmodbus/libmodbus-3.1.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+EAPI=7
+
+DESCRIPTION="Modbus library which supports RTU communication over a serial line or a TCP link"
+HOMEPAGE="https://libmodbus.org/"
+SRC_URI="https://libmodbus.org/releases/${P}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="static-libs test doc"
```
Comment (mgorny) [inline dev-libs/libmodbus/libmodbus-3.1.6.ebuild]: `RESTRICT="!test? ( test )"` below.

Also sort by flag name.

File: dev-libs/libmodbus/libmodbus-3.1.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+RDEPEND=""
+DEPEND="doc? ( app-text/asciidoc
+	app-text/xmlto )"
+
+PATCHES=( "${FILESDIR}"/${PN}-3.1.4-doc.patch )
+
+src_configure() {
+	econf \
+		--disable-silent-rules \
```
Comment (mgorny) [inline dev-libs/libmodbus/libmodbus-3.1.6.ebuild]: EAPI 7 already passes this for you.

File: dev-libs/libmodbus/libmodbus-3.1.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+		$(use_enable test tests) \
+		$(use_enable static-libs static) \
+		$(use_with doc documentation)
+}
+
+src_install() {
+	default
+
+	use static-libs || rm "${D}"/usr/*/libmodbus.la
```
Comment (mgorny) [inline dev-libs/libmodbus/libmodbus-3.1.6.ebuild]: Missing `||die`. Swap it to `if ! use ..` for readability.

Also, do static libs rreally require `.la` file? The package has no deps.

**Comment by juippis** [comment]
5dda1812d8300dc620b580bde40f57c88c0d0153

**Pull request closed without merging**